ZSSRforKernelGAN/ZSSR.py

```diff
@@ -305,8 +312,10 @@ def epoch_Z(self, hr_father, cropped_loss_map, kernel=None):
         else:
             # Final loss (Weighted (cropped_loss_map) L1 loss between label and output layer)
             loss_Disc = 0
+        if not self.norm_factor:
```

Comment:
loss_disc might be 0 if the flag self.conf.disc_loss is off. need to put that piece of code into the if statement